.github/workflows/runnable_cxx.yml

```
@@ -236,13 +236,12 @@ jobs:
         # On OSX, the system header are installed via `xcode-select` and not distributed with clang
         # Since some part of the testsuite rely on CC being only a binary (not a command),
         # and config files where only introduced from 6.0.0, use a wrapper script.
-        if [ "${{ matrix.os }}" == "macOS-10.15" ]; then
+        if [ "${{ matrix.os }}" == "macOS-11" ]; then
           # Note: heredoc shouldn't be indented
           cat << 'EOF' > ${TMP_CC}-wrapper
         #!/bin/bash
-        # Note: We need to use this because github.workspace is not stable
-        SCRIPT_FULL_PATH=$(dirname "$0")
-        ${SCRIPT_FULL_PATH}/clang -isystem /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/ $@
+        echo "Wrapper script called with: $@"
+        /usr/bin/cc $@
```

This is the _real_ fix for our custom github action environment
```suggestion
        # Note: We need to use this because github.workspace is not stable
        SCRIPT_FULL_PATH=$(dirname "$0")
        ${SCRIPT_FULL_PATH}/clang -isysroot $(xcrun --show-sdk-path) $@
```

**Or** add `export SDKROOT=$(xcrun --show-sdk-path)` somewhere.